lib/sql/get.js

```diff
@@ -143,6 +146,28 @@ class Get {
     let url = sql.sqlURL(this.url, this.db, this.appId, 'crud', `${this.table}/read`);
     return fetchAsync(url, this.options);
   }
+
+  apply(){
+    this.options.body = JSON.stringify(this.params);
+    let url = sql.sqlURL(this.url, this.db, this.appId, 'crud', `${this.table}/read`);
+    return fetchAsync(url, this.options);
+  }
+
+  op(op){
+    if(op === 'one'){
+      this.params.op = 'one';
```

Comment:
Just set the `this.params.op` to op received outside the if else. No if else is required here. If else will come in apply